From 6a41644c39ffb394c350de3dacbacdffa54a0a2a Mon Sep 17 00:00:00 2001
From: Adriaan de Groot <groot@kde.org>
Date: Sat, 3 Apr 2021 00:47:52 +0200
Subject: [PATCH] Fix CMake-time with CMake 3.20

Having multiple else-blocks for a single if() is now a syntax
error; they need to be elseif() with only a single
optional else() at the end.
---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 593dcb11..f965bd64 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -116,10 +116,10 @@ elseif(kst_3rdparty_download)
         elseif(GCC_VERSION VERSION_EQUAL 4.7 OR GCC_VERSION VERSION_GREATER 4.7)
             set(ver 4.7)
             set(md5 de6e8dbab1bb17eee6057941fddc93e3)
-        else(GCC_VERSION VERSION_EQUAL 4.6 OR GCC_VERSION VERSION_GREATER 4.6)
+        elseif(GCC_VERSION VERSION_EQUAL 4.6 OR GCC_VERSION VERSION_GREATER 4.6)
             set(ver 4.6)
             set(md5 70d8670af9c21eb8fb466654c95d8e4d)
-        else(GCC_VERSION VERSION_GREATER 4.4)
+        elseif(GCC_VERSION VERSION_GREATER 4.4)
             set(ver 4.4)
             set(md5 999248fb40a44543af4dd4cd1be0ceeb)
         else()
-- 
GitLab

